src/models/model.cpp

```diff
@@ -262,7 +262,7 @@ void EnsureDeviceOrtInit(OrtSession& session, DeviceType type) {
   if (device)
     return;
 
-  static const char* device_type_names[] = {"CPU (Not used, see above)", "Cuda", "DML", "WebGPU_Buffer", "QnnHtpShared"};
+  static const char* device_type_names[] = {"CPU (Not used, see above)", "Cuda", "DML", "WebGPU_Buffer", "QnnHtpShared", "Cpu"};
```

Comment:
Is OpenVINO just using the CPU allocator? If so, it should just be using the CPU device type. This is only for when it has a separate Ort::Allocator.

Reply:
Can you elaborate a bit here? Are you saying that we should have something other than `"Cpu"` for index 5 (when device=OpenVINO)?